native/src/components/List.tsx

```diff
@@ -11,10 +11,12 @@ const NoItemsMessage = styled.Text`
 
 type ListProps<T> = {
   items: Array<T>
-  noItemsMessage: ReactElement | string
+  noItemsMessage?: ReactElement | string
   renderItem: (props: { item: T; index: number }) => ReactElement
   Header?: ReactElement
   Footer?: ReactElement
+  scrollEnabled?: boolean
+  accessibilityRole?: AccessibilityRole | undefined
```

Comment:
No need to pass the `accessibilityRole` here as a prop. This component has a FlatList inside, which should always have the `accessibiltyRole="grid"`, see [RN docs](https://reactnative.dev/docs/accessibility#accessibilityrole). This will also add the "the in/out of grid announcements to the android GridView.", e.g. "four elements in the list"